crates/wasmtime/src/runtime/module.rs

```diff
@@ -348,37 +370,29 @@ impl Module {
                     &state,
 
                     // Cache miss, compute the actual artifacts
-                    |(engine, wasm)| -> Result<_> {
-                        let (mmap, info) = build_artifacts::<MmapVecWrapper>(engine.0, wasm)?;
-                        let code = publish_mmap(mmap.0)?;
-                        Ok((code, info))
+                    // Implementation of how to serialize artifacts
```

Comment:
This comment might be a copy/paste error?

Reply:
oops